.devcontainer/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM mcr.microsoft.com/devcontainers/typescript-node:0-16
+
+RUN npm install -g npm@9.6.4
```

Comment:
if node is there, then npm is there for sure
also docs for the base image mention npm https://github.com/devcontainers/images/blob/main/build/README.md?plain=1

is it necessary do to the installation of npm anyway?